tools/accuracy/README.md

```diff
@@ -0,0 +1,23 @@
+# AMD MIGraphX Accuracy checker
+## Instructions
+First ensure requirements and MIGraphX's python libary are installed. Refer to MIGraphX instructions at the root directory to install the python library.
```

Comment:
Add hyperlink to root directory instruction ? Also mention, it only works for GPU at the moment ?